Controller/ProcessController.php

```diff
@@ -150,8 +152,12 @@ public function new(Request $request, ProcessManager $processManager): Response
     /**
      * @Route("/{id}", name="process_show", methods={"GET", "POST"})
      */
-    public function show(Request $request, Process $process): Response
+    public function show(Request $request, Process $process, LoggerInterface $logger): Response
```

Comment:
What do you do with the LoggerInterface?

Reply:
Nothing :) Fixed